static/src/js/components/GranuleResultsHighlights/GranuleResultsHighlights.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+// TODO don't have map
```

Comment:
Remove this comment

Reply:
Yea I have a few of these throughout. Since there are so many icon swaps I wanted to call out places where we are deviating from their HDS design for people reviewing so they don't get confused why we still use Fa icon here. I will remove all these